src/pudl/ferc_to_sqlite/cli.py

```diff
@@ -96,7 +85,7 @@ def get_ferc_to_sqlite_job():
     "-w",
     "--workers",
     type=int,
-    default=0,
+    default=2,
     help=(
         "Number of worker processes to use when parsing XBRL filings. "
         "Defaults to using the number of CPUs."
```

Comment:
Is the default here still supposed to be 2? If so the help message needs to be updated.

Reply:
Let's keep the default as 0, which is what it was, and then force concurrency of 2 in `conftest.py`. Also, we set `--workers 8` in `gcp_pudl_etl.sh`, so that is unaffected by this default.